### DISCUSSION TITLE
[Script request]

### DISCUSSION THREAD

**Discussion by mdlmarkham** [community-scripts/ProxmoxVE#1080] [Request script]
### Application Name

Immich

### Website

https://immich.app/

### Description

High performance self-hosted photo and video management solution that is under "very active" development.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Use dockge and next time the search function.
